src/util/browserPolyfill.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @param {String} localeCode the 'xx-XX' language code for the app
+ * @return {String} the polyfill.io cdn string
+ */
+export function polyfillServiceUrl(localeCode) {
+	const features = [
```

Comment:
we should use a default set - `default-3.6` 

Then we can remove - 
Promise
URL
Array.from
String.prototype.endsWith
String.prototype.startsWith


docs - https://polyfill.io/v2/docs/features/#default-sets